{
	"schema_version": "1.6.0",
	"id": "BIT-tensorflow-2021-41205",
	"modified": "2024-03-06T11:25:28.861Z",
	"published": "2024-03-06T11:16:25.235Z",
	"aliases": [
		"CVE-2021-41205",
		"GHSA-49rx-x2rw-pc6f",
		"PYSEC-2021-398",
		"PYSEC-2021-615",
		"PYSEC-2021-813"
	],
	"details": "TensorFlow is an open source platform for machine learning. In affected versions the shape inference functions for the `QuantizeAndDequantizeV*` operations can trigger a read outside of bounds of heap allocated array. The fix will be included in TensorFlow 2.7.0. We will also cherrypick this commit on TensorFlow 2.6.1, TensorFlow 2.5.2, and TensorFlow 2.4.4, as these are also affected and still in supported range.",
	"affected": [
		{
			"package": {
				"ecosystem": "Bitnami",
				"name": "tensorflow",
				"purl": "pkg:bitnami/tensorflow"
			},
			"severity": [
				{
					"type": "CVSS_V3",
					"score": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:H/I:N/A:H"
				}
			],
			"ranges": [
				{
					"type": "SEMVER",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "2.4.4"
						},
						{
							"introduced": "2.5.0"
						},
						{
							"fixed": "2.5.2"
						},
						{
							"introduced": "2.6.0"
						},
						{
							"fixed": "2.6.1"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://github.com/bitnami/vulndb/tree/main/data/tensorflow/BIT-tensorflow-2021-41205.json"
			}
		}
	],
	"references": [
		{
			"type": "WEB",
			"url": "https://github.com/tensorflow/tensorflow/commit/7cf73a2274732c9d82af51c2bc2cf90d13cd7e6d"
		},
		{
			"type": "WEB",
			"url": "https://github.com/tensorflow/tensorflow/security/advisories/GHSA-49rx-x2rw-pc6f"
		}
	],
	"database_specific": {
		"cpes": [
			"cpe:2.3:a:google:tensorflow:*:*:*:*:*:*:*:*"
		],
		"severity": "High"
	}
}
